tldr/semantic.py

```
@@ -204,7 +242,19 @@ def get_model(model_name: Optional[str] = None):
             raise ValueError(f"Model download declined. Use --model to choose a smaller model.")
 
     from sentence_transformers import SentenceTransformer
-    _model = SentenceTransformer(hf_name)
+
+    logger.info(f"Loading model {hf_name} on device: {device}")
```

Ruff (0.14.14)

Logging statement uses f-string

(G004)